src/main/kotlin/org/rust/ide/inspections/fixes/RemovePathPrefixFix.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.inspections.fixes
+
+import com.intellij.codeInspection.LocalQuickFix
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.PsiFile
+import com.intellij.psi.SmartPointerManager
+import org.rust.openapiext.document
+
+/**
+ * Fix that removes a prefix of a certain path.
+ *
+ * E.g. `foo::bar::baz` -> `baz`.
+ */
+class RemovePathPrefixFix(
```

Comment:
Why not use `SubstituteTextFix.Companion#delete`?

Reply:
I wasn't aware that I can modify the fix name in such case. Thanks!